docs/proposals/14-wireguard-based-cluster-vpn.md

```diff
@@ -0,0 +1,178 @@
+
+# Cluster VPN based on Wireguard with reversed connection
+
+# Problem Statement
```

Comment:
Please rework the document to use [the given template](https://github.com/gardener/gardener/blob/master/docs/proposals/00-template.md) and read through the [proposal process description](https://github.com/gardener/gardener/blob/master/docs/proposals/README.md) carefully to familiarize yourself with the process.

Especially it would be good to clearly specify:
- goals and non-goals of the proposal
- alternatives considered

Reply:
I will add non-goals if I can think of any which is currently not the case. The document has an elaborate description of alternatives. Martin has added another nuance that I will add to the document (but not an alternative).